app/controllers/groups/layings_controller.rb

```diff
@@ -0,0 +1,41 @@
+module Groups
+  class LayingsController < ApplicationController
+    before_action :build_request, :data_check
+
+    ROUTE_MAP = {
+      index: proc { |params| ParliamentHelper.parliament_request.group_layings_index.set_url_params({ group_id: params[:group_id] }) }
+    }.freeze
+
+    def index
+      @group, @layings = FilterHelper.filter(@api_request, 'Group', 'Laying')
+      @group = @group.first
+
+      list_components = @layings.map do |laying|
+        laying_type = I18n.t('layings.type.si')
```

Comment:
Should this be in a conditional too, what if we release a different type of laid thing?